File: Weather-app/Models/CurrentWeatherData.m
//
//  CurrentWeatherData.m
//  Weather-app
//
//  Created by Reve on 11/14/22.
//

#import "CurrentWeatherData.h"

@implementation CurrentWeatherData

-(nullable instancetype) initWithDictionary: (nullable NSDictionary*) dict {
    if(self = [super init]) {
        self.time = dict[@"time"];
        self.temperature = dict[@"temperature"];
        self.windspeed = dict[@"windspeed"];
        self.winddirection = dict[@"winddirection"];
        self.weathercode = dict[@"weathercode"];
    }
    return self;
}
@end

File: Weather-app/Models/CurrentWeatherData.swift
//
//  CurrentWeatherData.swift
//  Weather-app
//
//  Created by Reve on 11/20/22.
//

import Foundation

class CurrentWeatherData: NSObject, Codable {
    let temperature: Double
    let windspeed: Double
    let winddirection: Double
    let weathercode: Int
    let time: String
}
